### ISSUE TITLE
Clarify CACHED dimension + add example + how to calculate cached percentage

### ISSUE THREAD

**Issue by rmarx** [rum-archive/rum-archive#27]
The `CACHED` dimension for the `resources` table is a bit weird, in that it uses boolean values in the histogram to indicate cached or not (i.e., values of `0` are not cached, `1` is cached). This seems to make it a bit awkward to work with in terms of percentiles and averages.

For example, this type of query works well for other dimensions that use histograms, but returns 0.0 for all averages and 1 for all medians:

```
SELECT
       URL as url,
       `akamai-mpulse-rumarchive.rumarchive.COMBINE_HISTOGRAMS`(ARRAY_AGG(CACHEDHISTOGRAM)) AS CachedHistogram,
       ROUND(SUM(CACHEDAVG) / SUM(FETCHES)) AS CachedPercentage,
        `akamai-mpulse-rumarchive.rumarchive.PERCENTILE_APPROX`(
          ARRAY_AGG(CACHEDHISTOGRAM),
          [0.5],
          100,
          false) as CachedMedian,
       COUNT(*) as rowcount,
       SUM(FETCHES) as fetchcount
       FROM `akamai-mpulse-rumarchive.rumarchive.rumarchive_resources`
       WHERE DATE = '2023-12-31'
       GROUP BY URL
       ORDER BY fetchcount DESC
```

Example output:

Row | url | CachedHistogram | CachedPercentage | CachedMedian | rowcount | fetchcount |  
-- | -- | -- | -- | -- | -- | -- | --
1 | https://www.googletagmanager.com/gtag/js | {"0":[0,9559165],"1":[1,3858984]} | 0.0 | 1 | 54767 | 13418149 |  
2 | https://www.googletagmanager.com/gtm.js | {"0":[0,6806158],"1":[1,1910448]} | 0.0 | 1 | 36756 | 8716606 |  
3 | https://www.facebook.com/tr/ | {"0":[0,5943209],"1":[1,1432831]} | 0.0 | 1 | 36730 | 7376040

For now, I've somewhat worked around this by writing a custom script that takes the `1` bucket from the histograms, divides it by the total fetchcount, to get the total Cached % for a given URL. However, this is annoying (is there no way to get a component of a histogram inside the query so I can do something like `histogram["1"] / SUM(FETCHES) as CachedPercentage` instead of needing a separate script?) and I'm not sure it's really correct. 

Long story short: `CACHED` is a weird dimension and we currently have no real documentation on how it's intended to work or how to get common insights from it (e.g., what does the CACHEDAVG and CACHEDSUMLN actually mean here)
